pkg/Cluster.go

```diff
@@ -64,7 +65,7 @@ func (c *Cluster) ServerVersion() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("%s.%s", info.Major, info.Minor), nil
+	return fmt.Sprintf("%s.%s", info.Major, strings.Trim(info.Minor, "+")), nil
```

Comment:
Should we use regex (\d+)([^\d]*) instead? this will make it agnostic to any other trailing characters also.

Reply:
What about parsing from `info.GitVersion` instead as I suggested in https://github.com/devtron-labs/silver-surfer/issues/5#issuecomment-929249098?

Seems like it will be more stable.

I don't have a GKE cluster to test on but they seem to be following a similar format: https://cloud.google.com/kubernetes-engine/versioning

So something like this:

```go
gitVersion := strings.Split(info.GitVersion, ".")
major := strings.Trim(gitVersion[0], "v")
minor := gitVersion[1]
return fmt.Sprintf("%s.%s", major, minor), nil
```

Reply:
Yes, gitVersion appears to be better indicator, lets use this instead